depends

```diff
@@ -1,7 +1,7 @@
 quilt
 parted
 realpath:coreutils
-qemu-arm-static:qemu-user-static
```

Comment:
Fix a bug in the check, sometimes maybe `qemu-arm-static` is available but `qemu-aarch64-static` is not.